ports/atmega168/apdu.c

```diff
@@ -41,6 +41,26 @@
 #include "bacnet/bacdcode.h"
 #include "bacnet/basic/services.h"
 
+static int Local_Network_Priority; /* Fixing test 10.1.2 Network priority */
```

Comment:
We should store the smaller uint8_t datatype that we're using in get/set.

Reply:
Fixed